hubble-be/src/main/java/com/baidu/hugegraph/controller/AboutController.java

```diff
@@ -22,40 +22,29 @@
 import java.util.LinkedHashMap;
 import java.util.Map;
 
-import org.apache.commons.io.FileUtils;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 import com.baidu.hugegraph.common.Constant;
 import com.baidu.hugegraph.handler.MessageSourceHandler;
-import com.baidu.hugegraph.license.LicenseVerifier;
-import com.baidu.hugegraph.util.Bytes;
 
 @RestController
 @RequestMapping("about")
+// TODO: delete the class or keep it?
 public class AboutController extends BaseController {
 
     @Autowired
     private MessageSourceHandler messageHandler;
 
     @GetMapping
     public Map<String, Object> about() {
-        LicenseVerifier verifier = LicenseVerifier.instance();
         Map<String, Object> about = new LinkedHashMap<>();
         about.put("name", Constant.SERVER_NAME);
         about.put("version", "1.5.0");
-        about.put("edition", verifier.edition());
-        about.put("allowed_graphs", verifier.allowedGraphs());
-        long datasize = verifier.allowedDataSize();
-        if (datasize == -1) {
-            about.put("allowed_datasize",
-                      this.messageHandler.getMessage("license.datasize.no-limit"));
-        } else {
-            about.put("allowed_datasize",
-                      FileUtils.byteCountToDisplaySize(datasize * Bytes.MB));
-        }
+        about.put("allowed_datasize",
```

Comment:
keep `about.put("edition", "community")` since used by UI
the same to `allowed_graphs`

Reply:
OK, try it later and add a screenshot

Reply:
keep remove it now (due to the ui is OK)